### PR TITLE
Allow additional ports for component services

### PR DESCRIPTION
**Pull request by maht**
Allow configuring extra ports for MQTT to ease proxy support.

The code is an adaptation of https://github.com/midokura/thingsboard-private/pull/687/files

See https://midokura.atlassian.net/browse/EVP-2568 for details.

### CODE REVIEW

File: aws/microservices/receipts/mqtt-load-balancer.yml [hunk 1]
```diff
@@ -20,4 +20,8 @@ spec:
     # This way NLB acts as transparent load balancer and forwards all traffic to port 8883 without decryption.
     - port: 8883
```
Comment (dmitry-midokura) [inline aws/microservices/receipts/mqtt-load-balancer.yml:21]: Can we add a value.yaml parameter for this port may be?

Comment (maht) [reply aws/microservices/receipts/mqtt-load-balancer.yml:21]: To my knowledge this YAML file is not part of the chart, but just a file given as reference for post-helm setup.

Comment (dmitry-midokura) [reply aws/microservices/receipts/mqtt-load-balancer.yml:21]: Oh, good catch :) So we are changing this file to provide an example to the operator? Why do we care about this file at all. Shall we only update the files that are part of helm chart?

Comment (maht) [reply aws/microservices/receipts/mqtt-load-balancer.yml:21]: Well, it turns out that now (contrary to EVP1 TB chart version) the services included in the Helm chart are not of load balancer type (by default default they are cluster IP type) so the are useless as a mean to provide extra external port for a proxy.

So we can change the chart so they can do it jointly with service type change to load balancer ... but probably the intention should be to not touch the chart defaults and instead install a cloud-provider specific recipe for the load balancer.

At least that is my impression (they include the recipes in this repo for a reason, aside of the Helm chart).

Comment (dmitry-midokura) [reply aws/microservices/receipts/mqtt-load-balancer.yml:21]: Ok, I lack context here. I don't know how these receipts are used. What I see is that MQTT service type is always LoadBalancer in EVP2 Azure deployments. That is why I was confused. Why don't we change the type in the helm chart if it needed for this task?

Comment (maht) [reply aws/microservices/receipts/mqtt-load-balancer.yml:21]: It turns out that we actually change it in the EVP2 chart values (see https://github.com/midokura/evp-chart/pull/274). I will remove the receipts commit.

**Review by xavileon — APPROVED** [review]

**Pull request merged**